assets/src/design-system/components/input/index.js

```diff
@@ -146,6 +147,10 @@ export const Input = forwardRef(
     const inputRef = useRef(null);
     const [focused, setFocused] = useState(false);
 
+    const isMultiple = value === MULTIPLE_VALUE;
+    value = isMultiple ? '' : value;
+    placeholder = isMultiple ? MULTIPLE_DISPLAY_VALUE : placeholder;
```

Comment:
@barklund @BrittanyIRL This PR brings the multiple value logic over from the editor since it's used generally. LMK if you think there might be issues with that.

Reply:
I really don't want this here. I would prefer just an `isIndeterminate` property (default `false`), that when set ignores `value` and just displays the placeholder.

I think the `MULTIPLE_VALUE` logic is pretty ugly and shouldn't go in the generalized API. I know that I invented it, but I still don't like it. It's _"impure"_.

Reply:
Wasn't it per UX to display `Mixed`?

Let's perhaps confirm with Sam and create a separate issue for replacing this logic everywhere? I can leave it in with a todo to remove. WDYT?

Reply:
If this is used in multiple places perhaps it would be better off as a variant or stored local to the editor in `edit-story/.../components/multipleValueInput/` where that logic can be stored and used specific to the editor without going into the design system?  

@samwhale added a numeric variant with a bit of logic in it, another option would be a multiple value input variant like that stored in the design system. I'm with @barklund that we should try to keep things in the ds generalized.

Reply:
Oh, we definitely need the _Mixed_ display, I just don't want the `MULTIPLE_VALUE` property to be part of it in this component. _Mixed_ is to be displayed, if `isIndeterminate` is true. The `MULTIPLE_VALUE` stuff must happen outside this component.

Reply:
Ah, ok, looks like I misunderstood what you meant.